src/hyperion/experiment_plans/tests/test_grid_detection_plan.py

```diff
@@ -158,9 +156,9 @@ def test_given_when_grid_detect_then_upper_left_and_start_position_as_expected(
     assert cb.out_upper_left[0] == [8, 2]
     assert cb.out_upper_left[1] == [8, 2]
 
-    assert gridscan_params.x_start == 0.0005
-    assert gridscan_params.y1_start == -0.0001
-    assert gridscan_params.z1_start == -0.0001
+    assert gridscan_params.x_start == 0.1
```

Comment:
Why did these values change?

Reply:
`gridscan_params = GridScanParams()` is located in fast_grid_scan.py. 
The values for `x_start`, `y1_start` and `z1_start` are defined in the `GridScanParams` class to begin with.
I change it to 0.1 since that is what the test is expecting in order to pass.
Is it wise to change the values of 0.0005, -0.0001, -0.0001 for `x_start`, `y1_start` and `z1_start` respectively in  `GridScanParams()`?

Reply:
No, that won't fix the problem.
Previously, the default parameter of `x_start = 0.1` in GridScanParams was modified in `grid_detection_plan` in line 167: `  out_parameters.x_start = start_positions[0][0]`, so the default value wasn't used.
What is different about the way `start_positions[0][0]` used to be calculated in `grid_detection_plan` compared to how `GridDetectionCallback` calculates `start_positions` now?